internal/services/sentinel/sentinel_alert_rule_anomaly_duplicate_resource_test.go

```diff
@@ -76,32 +76,54 @@ func TestAccSentinelAlertRuleAnomalyDuplicate_requiresImport(t *testing.T) {
 	})
 }
 
-func TestAccSentinelAlertRuleAnomalyDuplicate_withCustomObservation(t *testing.T) {
+func TestAccSentinelAlertRuleAnomalyDuplicate_ThresholdWithCustomObservation(t *testing.T) {
```

Comment:
Super minor, but test description are camelCased across the provider 
```suggestion
func TestAccSentinelAlertRuleAnomalyDuplicate_thresholdWithCustomObservation(t *testing.T) {
```